synapse/lib/stormtypes.py

```diff
@@ -6397,10 +6408,21 @@ async def _methCronAdd(self, **kwargs):
                 'incvals': incval,
                 'creator': self.runt.user.iden
                 }
+
         iden = kwargs.get('iden')
         if iden:
             cdef['iden'] = iden
 
+        view = kwargs.get('view')
```

Comment:
The current logic ( here and below ) would default the view to the user's default view but should probably set the default view to the current view that the command is running in.  Could probably eliminate this whole conditional block with
`kwargs.setdefault('view', self.runt.snap.view.iden)`

Reply:
:+1: